content/learn/manipulating-files/node-writing-files.en.md

```diff
@@ -66,33 +66,12 @@ fs.writeFile('/Users/joe/test.txt', content, { flag: 'a+' }, err => {});
 
 #### The flags you'll likely use are
 
-<table>
-  <tr>
-    <th>Flag</th>
-    <th>Description</th>
-    <th>File gets created if it doesn't exist</th>
-  </tr>
-  <tr>
-    <td><code>r+</code></td>
-    <td>This flag opens the file for <b>reading</b> and <b>writing</b></td>
-    <td style="text-align: center;">❌</td>
-  </tr>
-  <tr>
-    <td><code>w+</code></td>
-    <td>This flag opens the file for <b>reading</b> and <b>writing</b> and it also positions the stream at the <b>beginning</b> of the file</td>
-    <td style="text-align: center;">✅</td>
-  </tr>
-  <tr>
-    <td><code>a</code></td>
-    <td>This flag opens the file for <b>writing</b> and it also positions the stream at the <b>end</b> of the file</td>
-    <td style="text-align: center;">✅</td>
-  </tr>
-  <tr>
-    <td><code>a+</code></td>
-    <td>This flag opens the file for <b>reading</b> and <b>writing</b> and it also positions the stream at the <b>end</b> of the file</td>
-    <td style="text-align: center;">✅</td>
-  </tr>
-</table>
+|Flag|Description|File gets created if it doesn't exist|
```

Comment:
Usually, there are spaces between the `|`. You can check examples of Markdown tables on the web if you're in doubt. (https://www.tablesgenerator.com/markdown_tables)